// preset-ued@1.0.0/dist/rules.cjs
'use strict';

const _default = require('./shared/preset-ued.ac5ef8c2.cjs');
require('./shared/preset-ued.772cdc87.cjs');

const uedBorders = [
  ["border-radius-small", { "border-radius": "3px" }],
  ["border-radius-medium", { "border-radius": "6px" }],
  ["border-radius-large", { "border-radius": "9px" }],
  ["border-radius-circle", { "border-radius": "50%" }],
  ["border-width-small", { "border-width": "1px", "transform": "scale(0.5)" }],
  ["border-width-normal", { "border-width": "1px" }],
  ["border-width-medium", { "border-width": "2px" }],
  ["border-width-large", { "border-width": "3px" }],
  ["border-dash", { "border-style": "dash" }]
];

exports.alignments = _default.alignments;
exports.appearances = _default.appearances;
exports.aspectRatio = _default.aspectRatio;
exports.background = _default.background;
exports.boxSizing = _default.boxSizing;
exports.breaks = _default.breaks;
exports.color = _default.color;
exports.colorInteractionToken = _default.colorInteractionToken;
exports.colorToken = _default.colorToken;
exports.contains = _default.contains;
exports.contentVisibility = _default.contentVisibility;
exports.contents = _default.contents;
exports.cursors = _default.cursors;
exports.displays = _default.displays;
exports.flex = _default.flex;
exports.flexGridJustifiesAlignments = _default.flexGridJustifiesAlignments;
exports.floats = _default.floats;
exports.font = _default.font;
exports.fontShortcuts = _default.fontShortcuts;
exports.fontSmoothings = _default.fontSmoothings;
exports.fontStyles = _default.fontStyles;
exports.fontTypeMap = _default.fontTypeMap;
exports.fontWeight = _default.fontWeight;
exports.gaps = _default.gaps;
exports.insets = _default.insets;
exports.justifies = _default.justifies;
exports.margins = _default.margins;
exports.orders = _default.orders;
exports.overflows = _default.overflows;
exports.paddings = _default.paddings;
exports.placements = _default.placements;
exports.pointerEvents = _default.pointerEvents;
exports.positions = _default.positions;
exports.resizes = _default.resizes;
exports.rules = _default.rules;
exports.shadowToken = _default.shadowToken;
exports.sizes = _default.sizes;
exports.textAligns = _default.textAligns;
exports.textOverflows = _default.textOverflows;
exports.textTransforms = _default.textTransforms;
exports.textWraps = _default.textWraps;
exports.transformBase = _default.transformBase;
exports.transforms = _default.transforms;
exports.userSelects = _default.userSelects;
exports.varEmpty = _default.varEmpty;
exports.verticalAligns = _default.verticalAligns;
exports.whitespaces = _default.whitespaces;
exports.zIndexes = _default.zIndexes;
exports.uedBorders = uedBorders;
